src/lib/fal.ts

```diff
@@ -1,6 +1,9 @@
+"use client";
+
 import { createFalClient } from "@fal-ai/client";
 
 export const fal = createFalClient({
+  credentials: localStorage.getItem("falKey") as string,
```

Comment:
We should make this a function, so whenever the key changes this could reflect it:

```
credentials: () => localStorage.getItem("falKey")
```